moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -112,7 +136,7 @@ const ProjectNotes = (props) => {
   const [editingNoteType, setEditingNoteType] = useState(null);
   const [noteAddLoading, setNoteAddLoading] = useState(false);
   const [noteAddSuccess, setNoteAddSuccess] = useState(false);
-  const [editingNote, setEditingNote] = useState(false);
+  const [isEditingNote, setIsEditingNote] = useState(false);
```

Comment:
I independently came to this same edit in the branch I am working on! the state var names were confusing to me